certification/pyxis/pyxis.go

```diff
@@ -79,6 +84,45 @@ func (p *pyxisEngine) createImage(ctx context.Context, certImage *CertImage) (*C
 	return &newCertImage, nil
 }
 
+func (p *pyxisEngine) getImage(ctx context.Context, dockerImageDigest string) (*CertImage, error) {
+	req, err := p.newRequestWithApiToken(ctx, http.MethodGet,
+		getPyxisUrl(fmt.Sprintf("projects/certification/id/%s/images?filter=docker_image_digest==%s", p.ProjectId, dockerImageDigest)), nil)
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	resp, err := p.Client.Do(req)
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	defer resp.Body.Close()
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		log.Error(err)
+		return nil, err
+	}
+
+	if !checkStatus(resp.StatusCode) {
+		log.Errorf("%s: %s", "received non 200 status code in createImage", string(body))
+		return nil, errors.ErrNon200StatusCode
+	}
+
+	// using gjson to get the certImage we request since the payload is wrapped in an additional `data` element
+	certImage := gjson.Get(string(body), "data.0")
```

Comment:
Still unsure about introducing a new library for this.

Reply:
I added a local struct and removed that library. LMK what you think.